collector/collector.go

```diff
@@ -24,7 +24,15 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var namespace = "gnatsd"
+// System Name Varibles
+var (
+	// use gnatsd for backward compatiblity.  Changing would require users to
+	// change their dashboards or other applications that rely on the
+	// prometheus metric names.
+	CoreSystem       = "gnatsd"
```

Comment:
Should rename this one IMO.

Reply:
nats-server, nats-streaming

Reply:
A change would cause everyone who's created dashboards, alerts, etc to either reconfigure their exporters to set a prefix or alter their dashboards.